spget.py

```diff
@@ -34,7 +34,7 @@ def re_disamb(seq):
     wildcards = frozenset(IUPAC_WILDCARDS.keys())
     for c in seq:
         if c in wildcards:
-            out += "[" + IUPAC_WILDCARDS[c] + "]"
+            out += "[" + "|".join(IUPAC_WILDCARDS[c]) + "]"
```

Comment:
inside "[ ]" there's no need in "|" . Am I missing something?

Reply:
You're right, works both ways. Going to fix